cmds/core-service/main.go

```diff
@@ -44,6 +46,9 @@ var (
 	timeout    = flag.Duration("server timeout", 10*time.Second, "Default timeout for server calls")
 	locality   = flag.String("locality", "", "self-identification string used as CRDB table writer column")
 
+	systemIdentity = flag.String("system_identity", "", "System identity for which a version should be provided via the automated testing versioning interface")
+	systemVersion  = flag.String("system_version", "", "System version to provide via the automated testing versioning interface")
```

Comment:
I assume that, if the overall approach of the PR is correct, we will ultimately want this to return the same thing as the existing `aux/v1/version` endpoint. (obtained by a call to `version.Current()`.

(I'll update the PR once the open discussion points have been adressed)